packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -139,189 +122,24 @@ export default class QuanticSearchBox extends LightningElement {
       })) ?? [];
   }
 
-  /**
-   * @returns {quanticSearchBoxSuggestionsList}
-   */
-  get suggestionList() {
-    // @ts-ignore
-    return this.template.querySelector('c-quantic-search-box-suggestions-list');
+  get quanticSearchBoxInput() {
+    return this.template.querySelector('c-quantic-search-box-input');
   }
 
   /**
-   * @returns {HTMLInputElement|HTMLTextAreaElement}
+   * @returns {string}
    */
-  get input() {
-    return this.textarea
-      ? this.template.querySelector('textarea')
-      : this.template.querySelector('input');
+  get value() {
+    return this?.state?.value;
   }
 
   /**
-   * @returns {HTMLElement}
+   * @returns {Array}
    */
-  get combobox() {
-    return this.template.querySelector('.slds-combobox');
-  }
-
-  get searchBoxContainerClass() {
-    if (this.withoutSubmitButton) {
-      this.input?.setAttribute('aria-labelledby', 'fixed-text-label');
-      return CLASS_WITHOUT_SUBMIT;
-    }
-    this.input?.setAttribute(
-      'aria-labelledby',
-      'fixed-text-label fixed-text-addon-post'
-    );
-    return CLASS_WITH_SUBMIT;
-  }
-
-  get searchBoxInputClass() {
-    return `slds-input searchbox__input ${
-      this.withoutSubmitButton ? '' : 'searchbox__input-with-button'
-    }`;
-  }
-
-  get suggestionsOpen() {
-    return this.combobox.classList.contains('slds-is-open');
-  }
-
-  get isQueryEmpty() {
-    return !this.input?.value?.length;
-  }
-
-  showSuggestions() {
-    this.searchBox?.showSuggestions();
-    this.combobox?.classList.add('slds-is-open');
-    this.combobox?.setAttribute('aria-expanded', 'true');
-  }
-
-  hideSuggestions() {
-    this.combobox?.classList.remove('slds-is-open');
-    this.combobox?.setAttribute('aria-expanded', 'false');
-    this.suggestionList?.resetSelection();
-  }
-
-  handleHighlightChange(event) {
-    const suggestion = event.detail;
-    this.input.value = suggestion.rawValue;
-  }
-
-  handleEnter() {
-    const selectedSuggestion = this.suggestionList?.getCurrentSelectedValue();
-    if (this.suggestionsOpen && selectedSuggestion) {
-      this.searchBox.selectSuggestion(selectedSuggestion.rawValue);
-      this.input.blur();
-    } else {
-      this.searchBox.submit();
-      this.input.blur();
-    }
-  }
-
-  handleValueChange() {
-    if (this.searchBox.state.value !== this.input.value) {
-      this.searchBox.updateText(this.input.value);
-    }
-  }
-
-  onSubmit(event) {
-    event.stopPropagation();
-    if (this.searchBox.state.value !== this.input.value) {
-      this.searchBox.updateText(this.input.value);
-    }
-    this.searchBox.submit();
-    this.input.blur();
-  }
-
-  handleKeyValues() {
-    if (this.searchBox?.state?.value !== this.input.value) {
-      this.suggestionList?.resetSelection();
-      this.searchBox.updateText(this.input.value);
-    }
+  get suggestionsArray() {
```

Comment:
I can't use `suggestions` since it is already a duplicate name

<img width="1063" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/d58a49d9-f642-4804-8ebb-5824eaec2ad5">

Reply:
In this case you don't even need this getter, since it's just returning `this.suggestions`